Controller/CRUDController.php

```diff
@@ -361,11 +361,8 @@ public function editAction($id = null)
         $form->setData($object);
         $form->handleRequest($request);
 
-        if ($this->getRestMethod() == 'POST') {
-            $form->submit($request);
-
-            $isFormValid = $form->isValid();
-
+        $isFormValid = $form->isValid();
+        if ($form->isSubmitted() && $isFormValid) {
```

Comment:
This is not good because `if (!$isFormValid)` condition will never be reached.

You could at least use `if ($form->isSubmitted())` and put `$isFormValid = $form->isValid();` behind.